### DISCUSSION TITLE
how uninstall a skill

### DISCUSSION THREAD

**Discussion by geetee24** [OpenVoiceOS/ovos-buildroot#253] [Q&A] [answered]
https://openvoiceos.github.io/community-docs/081-ht_skills_install/

does not explain how to uninstall

**Comment by JarbasAl** [comment, marked as answer]
skills are regular python packages, so just `pip uninstall XXX`

if in doubt check the setup.py of the skill for the package name, eg https://github.com/OpenVoiceOS/skill-ovos-wikipedia/blob/dev/setup.py#L59
